todoman/ui.py

```diff
@@ -98,6 +99,14 @@ def __init__(self, todo, lists, formatter):
                         in widgets.ExtendedEdit.HELP)
         )
 
+        def change_current_list(radio_button, new_state, new_list):
+            self.current_list = new_list
+
+        for _list in self.lists:
+            urwid.RadioButton(items, _list.name, state='first True',
```

Comment:
Could you change the state to `_list = self.current_list` (instead of `"first True"`)? We need the current list selected by default.